MT/VR-MT-0006/VR-MT-0006.json

```diff
@@ -0,0 +1,94 @@
+{
```

Comment:
This file should be called `rule.json`. Right now, it's not picked up by the tooling.